front-end/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL=http://127.0.0.1:8000/api/v1
```

Comment:
Dont push your .env for security purposes
Just create and push .env.example then have its values blank if it is confidential.

Reply:
Thank you and sorry about this sir chev I forgot to add this in .gitignore, I'll re-push now.